src/App.jsx

```diff
@@ -1,10 +1,27 @@
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import Home from './views/Home/Home';
+import Login from './views/Login/Login';
 import LayoutViews from './views/LayoutViews/LayoutViews';
+import { PrivateRoute } from './components/PrivateRoute/PrivateRoute';
+import { ProvideAuth } from './context/ProvideAuth';
 
 export default function App() {
   return (
-    <LayoutViews>
-      <Home />
-    </LayoutViews>
+    <>
```

Comment:
Nice job here wrapping your context provider around your entire app to make state available to all of your files